nikola/data/themes/base/templates/base_helper.tmpl

```diff
@@ -42,7 +42,7 @@ lang="${lang}">
     % if meta_generator_tag:
         <meta name="generator" content="Nikola (getnikola.com)">
     % endif
-    ${feeds_translations.head()}
+    ${feeds_translations.head(classification=None, kind=kind, other=False)}
```

Comment:
There was actually no need for using `kind`, since `classification` is set to `None`. I've now set `kind` to `'index'`, which generates the same result.